packages/video_player/video_player_web/lib/video_player_web.dart

```diff
@@ -218,6 +224,11 @@ class _VideoPlayer {
   }
 
   void setVolume(double value) {
+    if (value > 0.0) {
```

Comment:
might need to introduce a small epsilon when comparing float values

Reply:
Or we make the `muted` property public so user can directly set mute instead of setting a 0.0 value.

Reply:
This "hatch" is only there so web users can do: setVolume(0.0) and get a "muted" player.

About making the `muted` property public: I agree, but I am not sure what the implications are for Android and iOS of that.

There's a huge API surface for the `video` element that is being hidden by the current platform API, and I'm not sure if it's because we want a very simple API, or because the API can't be extended for other platforms :/

For this ticket I was only trying to make the selected video auto-play in the image_picker web example, not do any significant changes to the API of the video_player widget.

Reply:
I've created this issue so we can discuss this new API: 

* https://github.com/flutter/flutter/issues/60721

If we decide to add a "muted" attribute to the public API, this hatch can be removed!